docs/index.rst

```diff
@@ -0,0 +1,25 @@
+fasjson-client
+==============
+
+A python client library for the FASJSON API
+
+This client uses the bravado library to build dynamic api methods based
+on open-api specs (version 2.0): `https://github.com/Yelp/bravado`_
```

Comment:
This seems to be causing build problems. I suggest making the `bravado` word earlier in the paragraph a hyperlink to the project's page.